pkgs/tools/X11/xkb-switch/default.nix

```
@@ -1,24 +1,24 @@
-{ stdenv, fetchgit, cmake, libX11 }:
+{ stdenv, fetchFromGitHub, cmake, libX11, libxkbfile }:
 
 stdenv.mkDerivation rec {
   name = "xkb-switch-${version}";
-  version = "1.3.1";
+  version = "1.5.0";
 
-  src = fetchgit {
-    url = https://github.com/ierton/xkb-switch.git;
-    rev = "351c84370ad0fa4aaaab9a32817859b1d5fb2a11";
-    sha256 = "0ilj3amwidi7imjvi8hr62y7j8zl809r5xhs7kv816773x32gpxq";
+  src = fetchFromGitHub {
+    owner = "ierton";
+    repo = "xkb-switch";
+    rev = "da4ef22e362a0939556d20db50053ab631631adc";
```

This repo has tags https://github.com/ierton/xkb-switch/releases/tag/1.5.0
```suggestion
    rev = version;
```